client/src/pages/Settings.tsx

```diff
@@ -5,7 +5,9 @@ import Input from "../components/Input";
 import Button from "../components/Button";
 import { useNavigate } from "react-router-dom";
 import axios from 'axios';
-import './Settings.css';
+
+import '../styles/Button.css';
+import '../styles/Input.css';
```

Comment:
These imports are not needed as they are already handled in their respective component files.